clients/src/test/java/org/apache/kafka/clients/consumer/internals/AsyncKafkaConsumerTest.java

```diff
@@ -844,6 +849,54 @@ public void testWakeupCommitted() {
         assertNull(consumer.wakeupTrigger().getPendingTask());
     }
 
+    @Test
```

Comment:
what's the interceptor behavior on close? if we have inflight commits before closing the consumer, should the interceptors be invoked? can we add tests around that?

Reply:
Added a test. Interceptors are called.